setup-r/src/installer.ts

```diff
@@ -277,6 +277,16 @@ async function acquireRtools(version: string) {
     rtools4 ? "rtools%s-x86_64.exe" : "Rtools%s.exe",
     version
   );
+
+  // If Rtools is already installed just return, as there is a message box
+  // which hangs the build otherwise.
+  if (
+    (!rtools4 && fs.existsSync("C:\\Rtools")) ||
```

Comment:
I don't remember what the default install directory for Rtools 35 is, but this is the location that `setup-r` would install it in.

In any case it seems they are only installing rtools40 on the images by default.